tasks/azure.py

```diff
@@ -0,0 +1,145 @@
+from ast import literal_eval
```

Comment:
Useful command line commands to interact with Azure

Reply:
Nice, looks good